src/lib/testing/expect-bound-function.ts

```diff
@@ -42,7 +52,12 @@ interface RunStage {
 
 const NO_RETURN_VALUE = Symbol('NO_RETURN_VALUE');
 
-class BoundFunctionTest<State, Args extends any[], Return> {
+class BoundFunctionTest<State, Args extends any[], Return>
+  implements
+    CallStage<State, Args, Return>,
+    WithMocksStage<State, Args, Return>,
+    WithReducerStage<State, Args, Return>,
+    AssertionStage<State, Args, Return> {
```

Comment:
Just as a suggestion: Let's order the interfaces here after the order they should be used.

Reply:
Done